src/Pages/Repositories/ContentListTable/components/AddContent/AddContent.tsx

```diff
@@ -500,7 +500,7 @@ const AddContent = ({ isEdit = false }: Props) => {
                   isDisabled={isEdit && contentOrigin === ContentOrigin.UPLOAD}
                   description={
                     isUploadRepo
-                      ? 'Create a repository to upload custom content to, snapshots will be taken after every new upload.'
+                      ? 'Create a repository to upload custom content to. Snapshots will be taken after every new upload, allowing you to build images with uploaded content.'
```

Comment:
This is the change for 4718, to add some information about image building to the upload option in the add/edit content modal

Reply:
ACK, that works:

![image](https://github.com/user-attachments/assets/6043a32b-d36b-486d-995d-44fb23483035)